### ISSUE TITLE
Add pywin32==225 to requirements (at least when using Python 3.8)?

### ISSUE THREAD

**Issue by charlie9578** [NREL/OpenOA#141]
I get the following error when trying to run an example notebook having followed the install guidelines. I suspect this is a general problem with python 3.8, https://github.com/mhammond/pywin32/issues/1431, but may be worth adding pywin32==225 to the requirements file so others don't hit this error...

 import win32api
ImportError: DLL load failed while importing win32api: The specified module could not be found.

The solution is to use: pip install --upgrade pywin32==225

**Comment by jordanperr** [comment]
Hi @charlie9578, thank you so much for reporting this issue. We are not experiencing this issue with our [Windows CI pipeline](https://github.com/NREL/OpenOA/runs/2196267929?check_suite_focus=true). I have read the thread on the pywin32 issue tracker, and it looks like this may be a specific issue with Python from the Microsoft Store instead of from the python.org installers. Could you please provide some more information about your development environment so I can try and re-create the problem. For example, Python version (and vendor), Windows version, Conda version, Pip version, and the steps you've taken to install OpenOA.

**Comment by charlie9578** [comment]
Hmmm, interesting... I tried to repeat the error in a new environment and it doesn't occur. So perhaps a freak event, or one I'd need to re-create with a full reinstall of Anaconda.

Either way: Python 3.8.8 (default, Apr 13 2021, 15:08:03) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32; MS Windows 10 Home, 10.0.19041 Build 19041; conda 4.9.2; pip 21.0.1. 

I used "conda create --name openoa-env python=3.8," cloned the OpenOA repo using GitHub desktop, navigated to use "pip install ./OpenOA," installed jupyter notebook "conda install -c conda-forge jupyterlab," ran "jupyter notebook," and then upon running the example notebook the error occured.
